index.js

```diff
@@ -83,6 +83,11 @@ const create = (win, options) => {
 				}
 			}),
 			separator: () => ({type: 'separator'}),
+			services: () => ({
+				id: 'services',
+				role: 'services',
+				visible: props.isEditable || hasText
```

Comment:
Services should work on more than just text. Can you try it with images and other things?

Reply:
I agree in principle, and I was surprised too – but I haven't been able to find anything other than text for which a services menu is shown for me in any app, except for the Finder which shows it for all files.

It could be that I don't have any apps that provide the appropriate services for, say, an image on a web page or in a Pages document, as opposed to an image file in the Finder? But if so, I'm not sure what apps might provide those services…

Of course, if the Electron implementation is sorted out it will presumably filter for content which has available services, and not show the menu item if no services are available, in which case we won't want any conditions on visibility here.

Reply:
Hmm, yeah. I have a lot of different services and I could not get any of them show up when right-clicking an image in Safari or Chrome, so seems like a weird system limitation. So I guess we can keep this as-is.